product_barcode_constraint_per_company/tests/test_module.py

```diff
@@ -1,37 +1,46 @@
-# Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+# Copyright 2023 Cetmix OÜ
+# Copyright (C) 2020-Today: GRAP (http://www.grap.coop)
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from psycopg2 import IntegrityError
 
-from odoo.tests.common import TransactionCase
+from odoo import _
+from odoo.tests.common import SavepointCase, tagged
 from odoo.tools.misc import mute_logger
 
 
-class TestModule(TransactionCase):
-    def setUp(self):
-        super().setUp()
+@tagged("post_install", "-at_install")
+class TestModule(SavepointCase):
```

Comment:
Please add assertion message for all assert cases and docstring for all tests functions